db/dev_seeds.rb

```diff
@@ -872,6 +871,17 @@ def unique_document_number
   end
 end
 
+section "Geolocating Proposals"  do
+  proposals = Proposal.select(:id).map{|p| p.id}
+  (1..([*1..Proposal.count].sample)).each do |i|
+    proposal = proposals.delete(proposals.sample)
+    MapLocation.create(latitude: 40.4167278 + rand(-10..10)/100.to_f,
```

Comment:
We'll be trying to provide data that is usable across all forks, so instead of hardcoding the latitude and longitude values... it would be better to use  `Setting['map_latitude']` && `Setting['map_longitude']` values along with `Setting['map_zoom']`